includes/Checker/Checks/General/I18n_Usage_Check.php

```diff
@@ -41,11 +41,27 @@ public function get_categories() {
 	 * @return array An associative array of PHPCS CLI arguments.
 	 */
 	protected function get_args() {
-		return array(
-			'extensions' => 'php',
+		$sniff_args = array(
 			'standard'   => 'WordPress',
+			'extensions' => 'php',
 			'sniffs'     => 'WordPress.WP.I18n',
 		);
+
+		global $argv;
+		$slug_prefix = '--force-slug=';
+		$result      = array_filter(
```

Comment:
I not liking this approach of fetching global arguments inside the check. May be we should save this new slug globally with which we can access from any checks. We need this value in other checks also.

Reply:
Ok, How could I make it?